### ISSUE TITLE
Misleading 2D Data Table image in README.md (2d_datatable.png)

### ISSUE THREAD

**Issue by redbar0n** [sajacy/google-sheets-what-if-analysis#13]
The readme image illustrating an example of a 2D (bivariate) Data Table is this:
https://github.com/sajacy/google-sheets-what-if-analysis/blob/master/doc/2d_datatable.png

That image is misleading, since that example won't actually work in practise, since the column headers need to be numeric values and not text. 

This description from the readme is correct according to how the script works, however:

> For example, to create a 2-dimensional (bivariate) Data Table, you need to select a range where the model output (formula) is the top-left cell, and where the two sets of values are provided along the first row and first column.

The row input and col input cells are not shown either. It would be nice to show them, so that showing what the script requires is complete. Otherwise, it may be confusing to use the script.

I would suggest including an example such as this instead:

| **2D (bivariate) Data Table** |    |    |    |    |
| --------------- | -- | -- | -- | -- |
| rowinput       | 2  |    |    |    |
| colinput       | 10 |    |    |    |
| model output  (=rowinput+2*colinput):          | **22** | 10 | 20 | 30 |
|                 | 1  | 21 | 41 | 61 |
|                 | 2  | 22 | 42 | 62 |
|                 | 3  | 23 | 43 | 63 |
|                 | 4  | 24 | 44 | 64 |

Or at least just update the PNG image with the slight alteration of adding numeric column header values so that it would make it a working example?

Update:
I have fixed the examples in Readme.md in the following commit, which I can make a pull request for once my outstanding pull request is accepted.

Changes to Readme.md: https://github.com/redbar0n/google-sheets-what-if-analysis/commit/6c7eb1cb88c3f3652ba53e6145744badf031264a
Preview: https://github.com/redbar0n/google-sheets-what-if-analysis

**Comment by sajacy** [comment]
I think in the code (which I merged but for which I just added a bug-fix), you inconsistently iterated over the `colinput`.  in the univariate case, you iterated "down" the column, but in the bivariate case, you iterated "across" the top row.  Similarly for `rowinput`, you iterated "down" in the bivariate case.  I've changed the words to avoid "rows" and "columns" and instead refer to "first input" and "second input" which corresponds to the left-column and the top-row, respectively. 

Thanks for pointing this out and adding the example - but I think the README can be updated to avoid "rowinput" and "colinput"?
